docs/source/models/concept_models/index.rst

```diff
@@ -4,10 +4,32 @@
 Simulation Concept Models
 =========================
 
+Active concept model documents can be found here:
+
 .. toctree::
    :maxdepth: 1
-   :glob:
 
-   */concept_model*
+   vivarium_census_synthdata/concept_model
+   vivarium_nutrition_optimization/concept_model
+   vivarium_us_cvd/concept_model
+   vivarium_wasting_paper/concept_model   
+
+Archive concept model documents can be found here:
+
+.. toctree::
+   :maxdepth: 1
```

Comment:
I'm curious whether it would "just work" if you made this second toctree using the original glob. I believe Sphinx is smart and won't put the same thing in two toctrees, so I _think_ it wouldn't include any of the active ones explicitly listed above?

If this works, it would make this shorter and also prevent us from accidentally forgetting one. If you want to control the order, we could add the glob **in addition** to the ones you have listed here.

Reply:
OK, so I tested that and it just duplicated the models listed in the first section. I think this is actually preferable though because this way the only risk is that we don't add a model to the top call-out section rather than leave anything out on accident. Now the two sections are (1) active models, and (2) all models (active and completed). Let me know if you think you preferred it the other way!

Reply:
Yeah that sounds good to me!